.clang-format

```diff
@@ -0,0 +1,75 @@
+---
+Language: Cpp
+
+# Indentation
+UseTab: Never
+IndentWidth: 2
```

Comment:
Shouldn't we stick to 4? I believe this is what's being used in cruby and generally the expected value for C code?

Reply:
Most of the current code uses 2. But if @soutaro is happy to update them all at once I'll be happy to update it too.

Reply:
Let's move to 4 to align the codebase with cruby. 👍